.github/workflows/dev-deploy.yaml

```
@@ -79,40 +58,93 @@ jobs:
+            - name: Run tests with coverage
+              run: |
+                  PYTHONPATH=$PWD poetry run pytest tests/ \
+                    --cov-branch \
+                    --cov=cloud_functions \
+                    --cov-report=xml \
+                    --cov-report=term \
+                    --junitxml=junit.xml \
+                    -o junit_family=legacy
+
+            - name: Upload coverage to Codecov
+              uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)